ckanext/canada/templates/internal/help.html

```diff
@@ -71,7 +70,6 @@ <h3>{{ _('Proactive Disclosure Information') }}</h3>
       <ol>
         <li>{{ _("Download the Latest Template to ensure that you're using the most up-to-date format.") }}</li>
         <li>{{ _('Prepare Data in the Proper Template: Organize your data in the appropriate template for the specific organization and Proactive publication type. Each template corresponds to particular context, so choose the relevant one.') }}</li>
-        <li>{{ _('Maintain the Abbreviated Organization Name: In the template, the abbreviated name of the organization (e.g., "aandc-aadnc") must remain in cell C1 exactly as downloaded.') }}</li>
         <li>{{ _("Include All Required Fields: Ensure that the template you're uploading contains all the required fields for the specific Proactive publication type. Missing fields can lead to issues during the upload process.") }}</li>
         <li>{{ _('Enter Valid Data: Populate the template with valid and accurate data. Incorrect or incomplete information may cause errors.') }}</li>
         <li>{{ _('File Format and Naming: Your completed data template should be an Excel spreadsheet in either XLS or XLSX format. Do not rename the file, as the system relies on a unique ID associated with each template. Renaming it could lead to complications.') }}</li>
```

Comment:
File naming does not matter anymore (which I noted in my review of this content), but if you did not get updated text for this, then we can just ignore and go ahead I guess...